05-how-to-contribute-to-open-source.md

```diff
@@ -25,139 +29,293 @@ OpenSauced is a powerful tool for finding open source projects to contribute to.
 
-```
-git checkout -b your-branch-name
-```
+   You can create a new branch using the following command:
+
+   ```
+   git checkout -b YOUR-BRANCH-NAME
+   ```
 
-Replace "your-branch-name" with a descriptive name for your branch, such as "fix-bug-123" or "add-new-feature".
+   Replace "YOUR-BRANCH-NAME" with a descriptive name for your branch, such as "fix-bug-123" or "add-new-feature".
 
-4. **Make your changes**: Now that you have a new branch, you can make changes to the codebase. Be sure to follow the project's coding guidelines and conventions, and test your changes to ensure they work as expected.
+4. **Make your changes**: Now that you have a new branch, you can make changes to the codebase. Always follow the project's coding guidelines and conventions, and test your changes to ensure they work as expected.
 
-5. **Commit your changes**: Once you've made your changes, stage and commit them using the following commands:
+5. **Add and commit your changes**: Once you've made your changes, add your changes to the staging area and commit them with these commands:
 
-```
-git add .
-git commit -m "Your commit message"
-```
+   ```
+   git add .
+   git commit -m "Your commit message"
+   ```
 
-Replace "Your commit message" with a brief description of the changes you made.
+   Replace `"Your commit message"` with a brief description of your changes.
 
 6. **Push your changes**: Push your changes to your forked repository on GitHub by running the following command:
 
-```
-git push origin your-branch-name
-```
+   ```
+   git push origin YOUR-BRANCH-NAME
+   ```
 
-Replace "your-branch-name" with the name of your branch.
+   Replace "YOUR-BRANCH-NAME" with the name of your branch.
 
-7. **Submit a pull request**: Once you've pushed your changes, navigate to the original project's repository on GitHub and click the "New pull request" button. In the "base repository" dropdown, select the original repository, and in the "head repository" dropdown, select your forked repository. Choose your branch from the "compare" dropdown, and click "Create pull request". Fill in the required information and submit your pull request.
+7. **[Create a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)**: Once you've pushed your changes, navigate to the original project's repository on GitHub and click the "New pull request" button. In the "base repository" dropdown, select the original repository, and in the "head repository" dropdown, select your forked repository. Choose your branch from the "compare" dropdown, and click "Create pull request". Fill in the required information and submit your pull request.
```

Comment:
They do stated on the GitHub's docs in the link, but I'm not sure if comparing base and head repositories are needed to be mentioned here since most of the time, we don't have to do that.

```suggestion
7. **[Create a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)**: Once you've pushed your changes, navigate to the original project's repository on GitHub and click the "New pull request" button. Fill in the required information and submit your pull request.
```